src/governance-vote.h

```diff
@@ -68,6 +68,8 @@ enum vote_signal_enum_t  {
     VOTE_SIGNAL_CUSTOM20   = 35
 };
 
+static const int MAX_SUPPORTED_VOTE_SIGNAL = 4;
```

Comment:
The question of these non-standard signals is currently very unclear to me.

What is clear is that 12.1 will not handle them.

I'm rather skeptical that we will ever be able to effectively handle all these signals.  The number of potential votes would be enormous unless some constraints are imposed.

For now I think it's better to leave the signal definitions alone and keep this limit separate.